src/components/MobileMenu.tsx

```diff
@@ -0,0 +1,118 @@
+import styled from '@emotion/styled';
+import Link from 'next/link';
+import React, { FC } from 'react';
+import { useLockBodyScroll, useToggle } from 'react-use';
+import { CTA } from '.';
+import { MenuIcon } from './icons';
+
+const textAnimation = `
+  opacity: 0;
+
+  .visible & {
+    opacity: 1;
+    transition: opacity 500ms ease 100ms;
+  }
+`;
+
+const Menu = styled.div`
+  background-color: ${({ theme }) => theme.colors.workThumbnailBg};
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  transform: scale(0, 0);
+  transform-origin: top right;
+  transition: transform 100ms ease-in-out;
```

Comment:
```suggestion
  transition: transform 220ms ease-out, opacity 220ms ease-out;
```
I'm not loving this fix either, but I think it is improved just visually. I would appreciate someone else's opinion on making this animation a little smoother.